Classes/Typo3ConfigurationLoader.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace KayStrobach\Migrations;
+
+use Doctrine\Migrations\Configuration\Configuration;
+use Doctrine\Migrations\Configuration\Migration\ConfigurationLoader;
+use Doctrine\Migrations\Metadata\Storage\TableMetadataStorageConfiguration;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use TYPO3\CMS\Core\Core\Environment;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Log\LogManager;
+use TYPO3\CMS\Core\Package\PackageInterface;
+use TYPO3\CMS\Core\Package\PackageManager;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * Implements a doctrine/migration ConfigurationLoader with a TYPO3 specific flavour
+ * - Adds
```

Comment:
Seems there's something missing here, right?

Reply:
For sure.. If I recall correctly I changed a lots of things in this loader which I noted when I started to try to document it. And then forgot to come back to it.

By reading the code, I think this description would seem appropriate:

```
* - Configure table `doctrine_migrationstatus` as where to store our migration status
* - Adds relevant TYPO3 specific paths where to search migrations:
*   1. `var/migrations/` and
*   2. `Migrations/Mysql` paths inside all active extensions
```